LnkParse3/LnkException.py

```diff
@@ -0,0 +1,2 @@
+class LnkException(Exception):
+    pass
```

Comment:
(Not related to this line)
Do we have something left to do (except not failing on malformed samples)?